views/package.json

```diff
@@ -8,7 +8,7 @@
   },
   "license": "GPL-2.0",
   "dependencies": {
-    "@oat-sa/tao-test-runner": "^0.2.1",
-    "@oat-sa/tao-test-runner-qti": "^0.3.2"
+    "@oat-sa/tao-test-runner": "^0.4.0",
```

Comment:
let's remove all carrets and tilds for now

Reply:
yep